server/api/src/main/java/io/smallrye/graphql/api/Namespace.java

```diff
@@ -0,0 +1,17 @@
+package io.smallrye.graphql.api;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import io.smallrye.common.annotation.Experimental;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ ElementType.TYPE })
+@Documented
+@Experimental("Experimental namespaces on api class. Separate namespaces via '/' (example - admin/users)")
```

Comment:
I would move the second sentence into Javadoc. And add a hint that the class name is being used as the default `value`.